app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__ContentBlockCreator.res

```diff
@@ -402,13 +426,14 @@ let handleFileInputChange = (
           )
         : None
     | #VideoEmbed =>
-      switch (
-        FileUtils.isVideo(file),
-        FileUtils.hasValidSize(~maxSize=FileUtils.defaultVideoMaxSize, file),
-      ) {
+      let maxVideoSize = maxVideoSize(vimeoAccountPlan)
+      switch (FileUtils.isVideo(file), FileUtils.hasValidSize(~maxSize=maxVideoSize, file)) {
       | (false, true | false) =>
         Some("Invalid file format, please select an MP4, MOV, WMV, AVI or FLV file.")
```

Comment:
Let's remove FLV? That's _Flash Video_. I'm pretty sure _no one_ uses that anymore...